inscriptions/ISic000387.xml

```diff
@@ -185,14 +190,18 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la" resp="#JH">
+                <ab>
+                    <lb n="1"/>SAC
```

Comment:
given that it's described as a fragment, It hink this has to be
`<gap reason="lost" extent="unknown" unit="character"/><orig>SAC</orig><gap reason="lost" extent="unknown" unit="character"/>` 
and likewise for line 2 (and no need for a space between the two letters in line 2